docs/src/main/sphinx/connector/postgresql.md

```diff
@@ -19,7 +19,7 @@ PostgreSQL instances.
 
 To connect to PostgreSQL, you need:
 
-- PostgreSQL 11.x or higher.
+- PostgreSQL 12.x or higher.
```

Comment:
PostgreSQL 11 is EOL (November 9, 2023)

PostgreSQL 12 is EOL too but since this is minimum version that we want to support I guess we are fine